internal/gps/result.go

```diff
@@ -61,19 +65,45 @@ func WriteDepTree(basedir string, l Lock, sm SourceManager, sv bool) error {
 		return err
 	}
 
-	// TODO(sdboyer) parallelize
-	for _, p := range l.Projects() {
-		to := filepath.FromSlash(filepath.Join(basedir, string(p.Ident().ProjectRoot)))
+	projects := l.Projects()
+	worksz := len(projects)
+
+	workChan := make(chan LockedProject, worksz)
+	doneChan := make(chan error, worksz)
```

Comment:
ok, no problem, I didn't want to add too much complexity, didn't know to which extent you want this to be modified. I get your point, will do :)